Error: iconv conversion from cp-850 to UTF-8 failed

### system info

Haraka | Haraka.js — Version: 2.8.16
 --- | :---
Node | v6.12.0
OS | Linux dione 4.4.0-62-generic #83-Ubuntu SMP Wed Jan 18 14:10:15 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
openssl | OpenSSL 1.0.2g  1 Mar 2016
iconv | 2.23

### Expected behavior

No Problem with CP-850 Mails.

### Observed behavior

`[ERROR] [-] [core] iconv conversion from cp-850 to UTF-8 failed: Conversion from cp-850 to UTF-8//TRANSLIT//IGNORE is not supported.`

### Steps to reproduce

Receive mails with CP-850 encoding.

### Solution?

Maybe the iconv option CP-850 is wrong and should be CP850?

--- a/mailbody.js
+++ b/mailbody.js
@@ -246,7 +246,7 @@ class Body extends events.EventEmitter {
                     this.bodytext = converter.convert(buf).toString();
                 }
                 catch (e) {
-                    logger.logerror(`iconv conversion from ${enc} to UTF-8 failed: ${e.message}`);
+                    logger.logwarn(`iconv conversion from ${enc} to UTF-8 failed: ${e.message}`);
                     this.bodytext = buf.toString();
                 }
             }

